src/VisualStudio/IntegrationTest/TestUtilities/LegacyEditorConfigCondition.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Options.EditorConfig;
+using Roslyn.Test.Utilities;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities
+{
+    public sealed class LegacyEditorConfigCondition : ExecutionCondition
+    {
+        public override bool ShouldSkip => EditorConfigDocumentOptionsProviderFactory.ShouldUseNativeEditorConfigSupport;
+
+        public override string SkipReason => "Test is only supported with our legacy .editorconfig support";
```

Comment:
😨

Reply:
Once we have machines with updated MSBuild and project system bits I hope to do the same old/new doubling like you did for the intellisense integration tests. But I can't effectively write that (and have any confidence on it) until the VMs actually have the new bits.